extensions/amp-social-share/1.0/amp-social-share.css

```diff
@@ -20,6 +20,16 @@
   * See https://github.com/ampproject/amphtml/issues/4277 for more details.
   */
 
+amp-social-share:focus-within {
+  outline: #0389ff solid 2px;
+  outline-offset: 2px;
+}
+
+amp-social-share::part(button):focus {
+  outline: none;
+  outline-offset: 0;
```

Comment:
What's this for?

Reply:
The outline-offset makes it surrounding focused box, further or closer to the component.  This resets to the default of 0.

outline-offset: 2px
![image](https://user-images.githubusercontent.com/25626770/105074764-d0b2d400-5a56-11eb-875f-e3bfb95a4fa7.png)

outline-offset: 15px
![image](https://user-images.githubusercontent.com/25626770/105074865-f17b2980-5a56-11eb-916b-972b42806fa3.png)

Reply:
Ok. Understood. Thanks for the explanation. If I understand it right, we want offset because the buttons all have variable vendor-based background and outline might not blend well with that. Good to proceed.

Reply:
Exactly!  Additionally, the exact focus indication styling (including the 2px `outline-offset` was recommended by the a11y consultants who audited all of our components (per the original issue: #29561).